client/src/components/layout/footer.module.css

```
@@ -1,2 +1,141 @@
-footer {
+.siteFooter {
+
+    background-color: #2e2e2e;
+    color: #ffffff;
+    width: 100%;
+    height: auto;
+    padding: 32px;
+    box-sizing: border-box;
+    font-family: 'Segoe UI', sans-serif;
```

Requested Changes: Design specs (which I should have included in the original ticket) ask for Roboto font.
```suggestion
    font-family: 'Roboto', sans-serif;
```